### ISSUE TITLE
ceil() fails validation, although it's valid glsl

### ISSUE THREAD

**Issue by infinitesnow** [gfx-rs/naga#330]
Using `ceil()` in shader returns
```
wgpu error: Validation Error

Caused by:
    In Device::create_shader_module
    Entry point main at Compute is invalid: Function(Resolve(FunctionNotDefined { name: "ceil" }))
```

**Comment by pjoe** [comment]
glsl-in doesn't (yet) handle all builtin functions.

Should get started on those 😄

**Comment by Systemcluster** [comment]
This is implemented and also checks for number of arguments 👍

**Comment by kvark** [comment]
Fixed by #338
